oddt/shape.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import scipy.stats as stats
+
+
+def calculate_standard_usr(atoms):
+    """
+    Function to calculate USR (Ultrafast Shape Recognition) shape descriptor
+    Args:
+        atoms: 3D coordinates of all molecule's atoms
+
+    Returns:
```

Comment:
Docstrings need to be [numpy style](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)